pyston/conf.py

```diff
@@ -40,7 +40,8 @@
     'PDF_EXPORT_TEMPLATE': 'default_pdf_table.html',
     'FILE_SIZE_LIMIT': 5000000,
     'AUTO_REVERSE': True,
-    'ALLOWED_INCOMPLETE_UPDATE': True,
+    'PARTIAL_PUT_UPDATE': False,
+    'PARTIAL_RELATED_UPDATE': False,
```

Comment:
kvůi starým aplikacím dovoluji nastavit PARTIAL_PUT_UPDATE. Ale by default to bude false. Stejně tak PARTIAL_RELATED_UPDATE když měníte objekt přes jiný objekt tak je sporné zda se chovat jako k patch nebo put